src/transformers/modeling_utils.py

```diff
@@ -1749,7 +1773,7 @@ def __init__(self, nf, nx):
     def forward(self, x):
         size_out = x.size()[:-1] + (self.nf,)
         x = torch.addmm(self.bias, x.view(-1, x.size(-1)), self.weight)
-        x = x.view(*size_out)
+        x = x.view(size_out)
```

Comment:
Is that change absolutely necessary? Are we sure it's 100% backward compatible with previous versions of PyTorch?

Reply:
Good point!
So I checked, the older version for which a documentation is available (0.1.12_2) says for [view](https://pytorch.org/docs/0.1.12/tensors.html?highlight=view#torch.Tensor.view): 

> Parameters:	args (torch.Size or int...) – Desired size

Which seems to be exactly the same for [1.10:](https://pytorch.org/docs/1.10.0/generated/torch.Tensor.view.html?highlight=view#torch.Tensor.view)

> shape (torch.Size or int...) – the desired size

So I think it is okay. The reason for this is that when unpacking args instead of giving a list, it iterates over args, but args is actually a `torch.Proxy` during tracing, and cannot be iterated over easily, causing tracing to fail.

Reply:
Thanks for checking!